server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tservers/TabletServerResource.java

```diff
@@ -233,7 +233,7 @@ public TabletServerSummary getTserverDetails(
   }
 
   private static final int concurrentScans = Monitor.getContext().getConfiguration()
-      .getCount(Property.TSERV_READ_AHEAD_MAXCONCURRENT);
+      .getCount(Property.TSERV_SCAN_EXECUTORS_DEFAULT_THREADS);
```

Comment:
I see you are updating this for your changes but it makes me wonder if this value on the Monitor is deceiving?

Reply:
I fixed that.